app/build.gradle

```diff
@@ -131,11 +133,16 @@ dependencies {
 
     implementation 'io.sentry:sentry-android:5.5.2'
 
+    implementation 'androidx.test:core-ktx:1.4.0'
```

Comment:
Why `implementation` and not `testImplementation`?

Reply:
It throws a compile error because 'androidx.test.core.app' is not resolved so we can't import ApplicationProvider which gives us the application context